proto/reaction.proto

```diff
@@ -261,6 +261,7 @@ message Compound {
   // properties of the compound (e.g., molecular weight), heuristic estimates
   // of physical properties (e.g., ClogP), optimized geometries (e.g., through
   // DFT), and calculated stereoelectronic descriptors.
+  // Convention: also put stoichiometry ratio/coefficient here, see https://github.com/open-reaction-database/ord-schema/issues/683
```

Comment:
I'd suggest creating a new section in the readme called "conventions" and moving this comment there; that's a better place for notes like this.